styles/style.css

```diff
@@ -94,8 +103,36 @@ p {
 
 
 /* Buttons */
+button {
+  border-radius: 0.3rem;
```

Comment:
I really love the way you've made this style, it's professional looking and you've used CSS varibles and classes to avoid reuse of coe. Objective 3 complete!